docs/reference/widgets/app.rst

```diff
@@ -3,5 +3,36 @@ Application
 
 The app is the main entry point and container for the Toga GUI.
 
+Usage
+-----
+
+The app class is used by instantiating with a name, namespace and callback to a startup delegate which takes 1 argument of the app instance.
```

Comment:
done in https://github.com/pybee/toga/pull/102/commits/45a92233b0d194afaee80cde9129ae7d8230ecf3